python/ray/runtime_env/runtime_env.py

```diff
@@ -18,6 +19,51 @@
 from ray.util.annotations import PublicAPI
 
 logger = logging.getLogger(__name__)
+CHAR_ENCODING = "utf-8"
+
+
+@PublicAPI(stability="alpha")
```

Comment:
I feel this function is not used for a normal user (they only use `runtime_env={"secret_env_vars":...}`) so maybe `@DeveloperAPI` ?

Reply:
Thanks! I did not know the conventions we use, I just added this annotation because the check on the PR complained about it 😄 . Let me change it to `@DeveloperAPI`.

Reply:
Updated